python_ta/transforms/z3_visitor.py

```diff
@@ -1,11 +1,12 @@
 import astroid
-from astroid import AstroidError, Uninferable, nodes
+from astroid import AstroidError, nodes
 from astroid.transforms import TransformVisitor
 from astroid.util import safe_infer
 from z3.z3types import Z3Exception
 
+from python_ta.z3.z3_parser import Z3ParseException, Z3Parser
```

Comment:
In addition to the above comment, don't include the blank line below. This import is the same "kind" as the `..contracts` import.